pkg/asset/ignition/machine/node.go

```diff
@@ -13,18 +13,19 @@ import (
 // pointerIgnitionConfig generates a config which references the remote config
 // served by the machine config server.
 func pointerIgnitionConfig(installConfig *types.InstallConfig, rootCA []byte, role string) *ignition.Config {
+	source := func() *url.URL {
```

Comment:
@staebler, looks like this function approach was [yours][1] in #206.  Do you remember why you took this route?  It seems complicated for a string pointer, vs:

```go
source := fmt.Sprintf("https://api.%s:22623/config/%s", installConfig.ClusterDomain(), role)
```

[1]: https://github.com/openshift/installer/pull/206/files#diff-9c7b009caefc0db38bdc6b4d024d27a8R62

Reply:
I don't think that was authored by me. That was authored by @crawford.